lighthouse-core/config/default-config.js

```diff
@@ -105,6 +105,10 @@ const UIStrings = {
   'not automatically checked by Lighthouse. They do not affect your score but it\'s important that you verify them manually.',
   /** Title of the Best Practices category of audits. This is displayed at the top of a list of audits focused on topics related to following web development best practices and accepted guidelines. Also used as a label of a score gauge; try to limit to 20 characters. */
   bestPracticesCategoryTitle: 'Best Practices',
+  /** Title of the Trust & Safety group of the Best Practices category. Within this section are the audits related to trust and safety. */
+  safetyGroupTitle: 'Trust & Safety',
+  /** Title of the General group of the Best Practices category. Within this section are the audits that don't belong to a specific group. */
+  generalGroupTitle: 'General',
```

Comment:
```suggestion
  bestPracticesGeneralGroupTitle: 'General',
```

I like that the others prefix with the category, especially nice for a group titled `General` ;)